### ISSUE TITLE
MVP Navigation Menu based on Role

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho#810]
Before MVP is released, we need a separate nav menu for public and psu users (non-application users.) The current navigation menu should be seen by application users.

Non-application users (public and logged in PSU users) should the following items on a nav bar:
- Home
- About
- Browse Collections (faceted search for collections)
- Browse Works (faceted search for works)
- Bookmarks
- History
- Log Out (if applicable)

Is there a good way to have these menu items also appear for application users?

#788 is a future enhancement to this.
